taxi/views.py

```diff
@@ -87,3 +111,18 @@ class DriverListView(LoginRequiredMixin, generic.ListView):
 class DriverDetailView(LoginRequiredMixin, generic.DetailView):
     model = Driver
     queryset = Driver.objects.all().prefetch_related("cars__manufacturer")
```

Comment:
check your code according to [checklist](https://github.com/mate-academy/py-taxi-service-user-form-and-validation/blob/master/checklist.md)
Also, when you define queryset it's not obligatory to have models, since queryset will overload models, you can simply delete lane 112.